README.md

```diff
@@ -6,7 +6,7 @@ Introduction goes here.
 Installation
 ------------
 
-Make sure to have the extension [solidus_social](https://github.com/solidusio-contrib/solidus_social) installed before adding this one.
+Make sure to have the extension [solidus_social](https://github.com/solidusio-contrib/solidus_social) installed (migrations ran) before adding this one.
```

Comment:
In here what I intend to say is that the migrations are the important part of having installed `solidus_social`, do you consider it confusing?